build.gradle

```diff
@@ -8,10 +8,10 @@ plugins {
 }
 
 group = 'uk.ac.ebi.ena.webin-cli'
-version = '7.3.1'
+version = '8.0.0'
```

Comment:
same comment here. @dipayan1985 i think your branched is based on the 7.3.1 version of webin-cli.